### PR TITLE
Created a table in the README.md

### PR DESCRIPTION
**Pull request by seun-beta**
Signed-off-by: Seunfunmi Adegoke <51419148+seun-beta@users.noreply.github.com>

### CODE REVIEW

**Comment by seun-beta** [comment]
@GeorgLink I created the tables in the Risk Working README.md according to the template prescribed in the Standardization google document.

File: focus-areas/business-risk/README.md [hunk 1]
```diff
@@ -1,36 +1,13 @@
-# Business Risk (How likely am I to be stuck with this project and no community)
```
Comment (GeorgLink) [inline focus-areas/business-risk/README.md:1]: Please add the Focus Area as a title.

File: focus-areas/code-quality/README.md [hunk 2]
```diff
@@ -1,24 +1,7 @@
-# Code Quality
```
Comment (GeorgLink) [inline focus-areas/code-quality/README.md:1]: Please add the Focus Area as a title.

File: focus-areas/licensing/README.md [hunk 3]
```diff
@@ -1,27 +1,10 @@
-# Licensing
```
Comment (GeorgLink) [inline focus-areas/licensing/README.md:1]: Please add the Focus Area as a title.

File: focus-areas/security/README.md [hunk 4]
```diff
@@ -1,10 +1,8 @@
-# Security
```
Comment (GeorgLink) [inline focus-areas/security/README.md:1]: Please add the Focus Area as a title.

File: focus-areas/transparency/README.md [hunk 5]
```diff
@@ -1,8 +1,6 @@
-# Transparency
```
Comment (GeorgLink) [inline focus-areas/transparency/README.md:1]: Please add the Focus Area as a title.

**Review by GeorgLink — CHANGES_REQUESTED** [review]
Thanks @seun-beta 

Please add the title of the Focus Area at the top of each file.

I will leave the remaining review to more active WG members

File: focus-areas/business-risk/README.md (hunk 1 above)
Comment (seun-beta) [reply focus-areas/business-risk/README.md:1]: Resolved

File: focus-areas/licensing/README.md (hunk 3 above)
Comment (seun-beta) [reply focus-areas/licensing/README.md:1]: Resolved

File: focus-areas/transparency/README.md (hunk 5 above)
Comment (seun-beta) [reply focus-areas/transparency/README.md:1]: Resolved

File: focus-areas/security/README.md (hunk 4 above)
Comment (seun-beta) [reply focus-areas/security/README.md:1]: Resolved

File: focus-areas/code-quality/README.md (hunk 2 above)
Comment (seun-beta) [reply focus-areas/code-quality/README.md:1]: Resolved

**Comment by seun-beta** [comment]
Thank you very much @GeorgLink for the review. I would be on stand by for more reviews from more active WG members. 

Thanks!

**Comment by GeorgLink** [comment]
Thanks @seun-beta

**Comment by ritik-malik** [comment]
Hi @sgoggins ,
Is this PR ready to merge?
If yes, it would be of great help for us to proceed with the process of [standardization of Working Groups repos](https://docs.google.com/document/d/1chPzgJa49sO_f3wVqp_NLJupSVyKHSVyuFuwzl4m4KI/edit#heading=h.4ejkic2mb7z1), which serves as a prerequisite for our [M.A.R.S.](https://github.com/chaoss/MARS) project, since we would be pulling focus-area goals directly from their respective READMEs

**Comment by klumb** [comment]
@sgoggins where are we at on this?

**Pull request merged**